rails/app/assets/stylesheets/components/_balloon.scss

```diff
@@ -1,19 +1,17 @@
 .ts-markerPopup {
   &-content {
-    display: flex;
     justify-content: center;
```

Comment:
Since we are removing `display: flex` we should also remove this — this CSS attribute won't do anything without display being set to flex!

Reply:
It has been updated! All good to go